src/main/java/com/example/demo/repository/TeamRepository.java

```diff
@@ -0,0 +1,59 @@
+package com.example.demo.repository;
+
+import com.example.demo.model.Team;
+import com.example.demo.tournament.CSVcontroller;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class TeamRepository implements BaseTeamRepository {
+
+    private static final List<Team> teams;
```

Comment:
For the CLI it doesn't matter much, but for real applications(especially web) this approach won't be thread safe. Generally you would like to use thread safe collection here